redis/Makefile

```diff
@@ -0,0 +1,98 @@
+
+REDIS_VERSION:=3.0.7
```

Comment:
Wow, tough piece :) Did you write it from scratch? I suppose at least the ports should be pulled into variables/parameters?

All in all I feel we really need to look into using Docker for setting up stuff like that...